source/core/aggregation-pipeline.txt

```diff
@@ -194,41 +194,29 @@ pipeline.
 Considerations
 --------------
 
-Sharded Collections
-~~~~~~~~~~~~~~~~~~~~
+Aggregation Pipeline as an Alternative to Map-Reduce
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
• Same here, usage of agg pipelines on sharded cluster, limitations, and optimization seems more important than the MR replacement. Might this also be better suited as the last "Considerations" subheading?

Reply:
Done. Will rebuild stage and check changes.